hazelcast/src/main/java/com/hazelcast/map/IMap.java

```diff
@@ -863,6 +863,58 @@
                                 long ttl, @Nonnull TimeUnit ttlUnit,
                                 long maxIdle, @Nonnull TimeUnit maxIdleUnit);
 
+    /**
+     * Asynchronously puts the given map entry in the map store.
```

Comment:
Map Store is another feature of ours, and this sentence seem like talking about it, but actually is not. Can we remove the word `store` from here to avoid confusion ?

Reply:
Thanks for your comment. Sure. Shall I add the comment as "Asynchronously puts the given map entry to this map". Will this be ok?

Reply:
Entry is used for single key-value mapping in a map. So this sound like single entry, rather than a map. Can we say "Asynchronously copies all of the mappings from the specified map to this map" ? 
Similarly,  `@param map the map entry` should change to something like 
`@param map mappings to be stored in this map`

Reply:
Yes that makes sense. Will make the change. Thanks.

Reply:
I've now updated it.